### PR TITLE
issue-4 NBS-4829 [disk manager] add component and task id to dm logs

### PR DESCRIPTION
**Pull request by gy2411**
#4

### CODE REVIEW

File: cloud/tasks/logging/fields.go [hunk 1]
```diff
@@ -16,6 +16,14 @@ type Field = log.Field
...
 
 const ComponentYDB = "YDB"
+const ComponentS3 = "S3"
+const ComponentNbs = "NBS"
+const ComponentNfs = "NFS"
+const ComponentRunners = "RUNNERS"
+const ComponentScheduler = "SCHEDULER"
+const ComponentTask = "TASK"
+const ComponentApp = "APP"
+const ComponentClient = "CLIENT"
```
Comment (SvartMetal) [inline cloud/tasks/logging/fields.go:26]: Do we have to place "client", "nfs", "nbs" into "tasks" library?

Comment (SvartMetal) [reply cloud/tasks/logging/fields.go:26]: Lets add components only for tasks (for now)?

File: cloud/tasks/logging/fields.go [hunk 2]
```diff
@@ -16,6 +16,14 @@ type Field = log.Field
 ////////////////////////////////////////////////////////////////////////////////
 
 const ComponentYDB = "YDB"
+const ComponentS3 = "S3"
+const ComponentNbs = "NBS"
+const ComponentNfs = "NFS"
+const ComponentRunners = "RUNNERS"
```
Comment (SvartMetal) [inline cloud/tasks/logging/fields.go:22]: TASK_RUNNER

File: cloud/tasks/logging/fields.go [hunk 3]
```diff
@@ -16,6 +16,14 @@ type Field = log.Field
...
+const ComponentScheduler = "SCHEDULER"
```
Comment (SvartMetal) [inline cloud/tasks/logging/fields.go:23]: TASK_SCHEDULER

File: cloud/tasks/logging/fields.go (hunk 1 above)
Comment (gy2411) [reply cloud/tasks/logging/fields.go]: I agree that it is better not to place these components  into "tasks" library. Removed them, now there are no changes outside "tasks" in this pr.

Is it ok to add "client", "nfs", "nbs", "app" components in separate pr?

File: cloud/tasks/runner_test.go [hunk 4]
```diff
@@ -284,7 +284,7 @@ func TestRunnerForRun(t *testing.T) {
 		State:   []byte{2, 3, 4},
 	}
 	task.On("Save").Return(state.State, nil)
-	task.On("Run", ctx, execCtx).Return(nil)
+	task.On("Run", mock.Anything, execCtx).Return(nil)
```
[the remark this answers is not in the document]
Comment (gy2411) [reply cloud/tasks/runner_test.go:287]: Контекст, с которым вызовется метод Run, теперь отличается от ctx (т.к. в него перед вызовом Run добавлябтся новые поля). Поэтому тест в старой версии падает.

**Review by BarkovBG — APPROVED** [review]

**Review by SvartMetal — APPROVED** [review]

**Review by BarkovBG — APPROVED** [review]

**Pull request merged**